### DISCUSSION TITLE
Permissions / security for updating or deleting entries?

### DISCUSSION THREAD

**Discussion by mandrasch** [wpengine/atlas-content-modeler#363] [Q&A / Help / Support] [answered]
Hey, 

I just tried out SvelteKit with WPGraphQL, had much fun.
https://www.youtube.com/watch?v=94FZvB6B_c0

Afterwards I thought about using Atlas Content Modeler for a small hobby project. I want to have content types/models which are only editable by the user who created them. Is this already possible with Atlas Content Modeler? Could'nt find information about this in the docs.

Thanks very much in advance! 👋

**Comment by nickcernis** [comment, marked as answer]
Hi Matthias

Great to hear you had fun trying out SvelteKit with WPGraphQL! Thanks for exploring Atlas Content Modeler too.

ACM does not yet support restricting post type editing by user. Granular publisher permissions are something we've talked about, but I don't yet have a timeline for the feature.

There are paid WordPress plugins that let you limit which users can edit a post type (such as [PublishPress](https://publishpress.com/capabilities/)) if you wanted a way to implement this for now.

**Reply by mandrasch** [reply]
Hi @nickcernis!

Thanks very much for your quick reply and the suggestions! Good to know that I haven't missed anything (yet). 👍 

Best regards,
Matthias
